docs/src/pages/[platform]/getting-started/migration/migration.react.mdx

```diff
@@ -52,11 +52,22 @@ For customers with applications using right to left directionality, the example
 ```
 Also see: [Internationalization](react/getting-started/internationalization)
 
-#### 3. Legacy Authenticator exports removed
+#### 3. Legacy component exports removed
+
+Remove any of the following deprecated components imported from `@aws-amplify/ui-react/legacy` .
```

Comment:
Should we add a note if they want to mix old components with new ones they can do an npm alias?

Reply:
I'm not sure if they can, since the new version will require `aws-amplify` version `5.x`. I think they would get peerDepenency conflict.